backend-api/src/functions/testUtils/unitTestData.ts

```diff
@@ -46,15 +31,93 @@ export const validBaseSessionAttributes = {
   timeToLive: 12345,
 };
 
+export const invalidBaseSessionAttributeTypes = {
+  clientId: "mockClientId",
+  govukSigninJourneyId: "mockGovukSigninJourneyId",
+  createdAt: validCreatedAt,
+  issuer: 12345, // Invalid type
+  sessionId: mockSessionId,
+  sessionState: SessionState.AUTH_SESSION_CREATED,
+  clientState: "mockClientState",
+  subjectIdentifier: "mockSubjectIdentifier",
+  timeToLive: "12345", // Invalid type
+};
+
+// Biometric token issued
+
 export const validBiometricTokenIssuedSessionAttributes = {
   ...validBaseSessionAttributes,
   sessionState: SessionState.BIOMETRIC_TOKEN_ISSUED,
   documentType: "NFC_PASSPORT",
   opaqueId: "mockOpaqueId",
 };
 
+export const validBiometricTokenIssuedSessionAttributesMobileApp = {
```

Comment:
suggestion (non-blocking): 

```suggestion
export const validBiometricTokenIssuedSessionAttributes = {
```

Other than 'base' attributes, the other attribute sets have typically just followed a naming pattern based purely on the state, so I don't think it's that helpful to state `MobileApp` here

Entirely non-blocking, if you like it all good to resolve

Reply:
Curious if you did think of anything I'm missing about adding MobileApp

Reply:
As the values are different for MAM and DAD sessions, do you have a suggestion of how to distinguish them? No worries if not but as maybe appending `Mam` instead of `MobileApp` is better?

Reply:
Nope that makes sense, I didn't think of that, happy to leave it as is